src/components/Jobs.vue

```diff
@@ -92,7 +96,21 @@ export default {
       });
     },
     runSelectedJob() {
-      console.log('Run job: ' + this.runnableJob.selected.name);
+      let apiPath = this.runnableJob.selected.path;
+      this._.each(
+        this.runnableJob.selected.requiredParameters,
+        (requiredParameter, index) => {
+          apiPath = this._.replace(
```

Comment:
So `this._` is the usual way to inject Lodash into Vue?

Reply:
Yes, a big odd – see https://github.com/Ewocker/vue-lodash  There might be a better util and/or usage.